app/models/illiad_openurl.rb

```diff
@@ -0,0 +1,35 @@
+###
+#  Class to handle creation of ILLiad OpenURL request
+###
+class IlliadOpenurl
+  def initialize(scan, redirect_url)
+    @scan = scan
+    @redirect_url = redirect_url
+  end
+
+  def illiad_page
```

Comment:
I think you can safely drop the `illiad_` prefix from all these methods, since they are in an `Illiad` class anyway.